helm/alfresco-content-services/values.yaml

```diff
@@ -209,75 +209,53 @@ alfresco-transform-service:
   enabled: true
   messageBroker:
     existingSecretName: *acs_messageBroker_secretName
-  transformrouter: &ats-router
+  transformrouter:
+    # -- Declares the alfresco-transform-router service used by the content repository
+    # to route transformation requests
+    enabled: true
     replicaCount: 2
     image:
       repository: quay.io/alfresco/alfresco-transform-router
       tag: 3.0.0
-  # -- Deprecated, use alfresco-transform-service.pdfrenderer instead
-  pdfrenderer: &ats-pdf
+  pdfrenderer:
+    # -- Declares the alfresco-pdf-renderer service used by the content repository
+    # to transform pdf files
+    enabled: true
     image:
       repository: alfresco/alfresco-pdf-renderer
       tag: 4.0.0
-  # -- Deprecated, use alfresco-transform-service.imagemagick instead
-  imagemagick: &ats-im
+  imagemagick:
+    # -- Declares the alfresco-imagemagick service used by the content repository
+    # to transform image files
+    enabled: true
     image:
       repository: alfresco/alfresco-imagemagick
       tag: 4.0.0
-  # -- Deprecated, use alfresco-transform-service.libreoffice instead
-  libreoffice: &ats-lo
+  libreoffice:
+    # -- Declares the alfresco-libreoffice service used by the content repository
+    # to transform office files
+    enabled: true
     image:
       repository: alfresco/alfresco-libreoffice
       tag: 4.0.0
-  # -- Deprecated, use alfresco-transform-service.tika instead
-  tika: &ats-tika
+  tika:
+    # -- Declares the alfresco-tika service used by the content repository
+    # to transform office files
+    enabled: true
     image:
       repository: alfresco/alfresco-tika
       tag: 4.0.0
-  # -- Deprecated, use alfresco-transform-service.transformmisc instead
-  transformmisc: &ats-misc
+  transformmisc:
+    # -- Declares the alfresco-tika service used by the content repository
+    # to transform office files
+    enabled: true
     image:
       repository: alfresco/alfresco-transform-misc
       tag: 4.0.0
-  aiTransformer:
```

Comment:
because it was introduced by mistake during https://github.com/Alfresco/acs-deployment/pull/960 when moving stuff under the new `alfresco-transform-service` key